<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SettingsDialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Загальні</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Мова інтерфейсу:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="languageComboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Тема оформлення:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="themeComboBox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="closeButton_2">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>350</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Закрити</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Тестування</string>
      </attribute>
      <widget class="QCustomPlot" name="plotWidget" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>451</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="runBenchmarkButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>341</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Запустити тест</string>
       </property>
      </widget>
      <widget class="QLabel" name="resultLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>411</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>Результати будуть тут...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="closeButton">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>330</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Закрити</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Статистика</string>
      </attribute>
      <widget class="QCustomPlot" name="statsPlotWidget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>471</width>
         <height>331</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="closeButton_3">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>350</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Закрити</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton_3</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
